src/main/java/game/GameLoop.java

```diff
@@ -0,0 +1,45 @@
+package game;
+
+public class GameLoop {
+
+    //String emptyField = "         ";
+    String emptyField = "_________";
+    char symbol = 'X';
```

Comment:
These fields are not needed outside of the class, so make them private please. Also they are constant so make them final as well.